### ISSUE TITLE
The elucidation for `realizes` references a deprecated property

### ISSUE THREAD

**Issue by ajnelson-nist** [oborel/obo-relations#671]
Hello,

In reviewing the documentation for the `realizes` property (`obo:BFO_0000055`), I saw a reference to `bearer_of` (`obo:RO_0004097`).

[These lines in today's `core.owl`](https://github.com/oborel/obo-relations/blob/9c0bbac625ce7748403685cd50c07392096f732b/core.owl#L176-L189) contain the elucidation, which in Turtle reads like this:

```turtle
obo:BFO_0000055
	rdfs:label "realizes"@en ;
	obo:IAO_0000600 "to say that b realizes c at t is to assert that there is some material entity d & b is a process which has participant d at t & c is a disposition or role of which d is bearer_of at t& the type instantiated by b is correlated with the type instantiated by c. (axiom label in BFO2 Reference: [059-003])"@en ;
	.
```

[The full definition of `bearer_of` in today's `core.owl`](https://github.com/oborel/obo-relations/blob/9c0bbac625ce7748403685cd50c07392096f732b/core.owl#L616-L621) reads like this in Turtle:

```turtle
obo:RO_0004097
	a owl:ObjectProperty ;
	rdfs:label "DEPRECATED bearer of"@en ;
	owl:deprecated "true"^^xsd:boolean ;
	.
```

Are there plans to replace that "bearer of" reference in the elucidation for `realizes`?

**Comment by balhoff** [comment]
I'm not sure of the history of the RO:0004097 term, but RO contains another 'bearer of' term RO:0000053 which has the preferred label 'has characteristic'. Probably there should be a `replaced_by` link between them.

**Comment by wdduncan** [comment]
IMHO, I find the definition is quite wordy and hard to read. I would reword this section of the definition:

> which has participant d at t & c is a disposition or role of which d is bearer_of at t

using `characteristic of` (the inverse of `has characteristic`) like this:

> which has participant d at t & c is a disposition or role which a **characteristic of** d at t 

Thoughts?

**Comment by wdduncan** [comment]
I don't know how BFO-2020 relates to the OBO Foundry. There are a differences with BFO 2.0, and may OBO ontologies use the classes-only version of BFO 2.0 in conjunction with RO relations.
